tests/cpp/test_host_irs.cpp

```diff
@@ -708,6 +708,43 @@ INSTANTIATE_TEST_SUITE_P(
       return ss.str();
     });
 
+using MatmulHostIrTest = NVFuserTest;
+
+TEST_F(MatmulHostIrTest, HostIr) {
+  constexpr int64_t H = 32;
+  constexpr int64_t M = 64;
+  constexpr int64_t K = 128;
+  constexpr int64_t N = 256;
+
+  auto hic = std::make_unique<HostIrContainer>();
+  FusionGuard fg(hic.get());
+
+  TensorView* A = makeContigTensor(3);
+  TensorView* B = makeContigTensor(3);
+  TensorView* C = matmul(A, B);
```

Comment:
```suggestion
  TensorView* a = makeContigTensor(3);
  TensorView* b = makeContigTensor(3);
  TensorView* c = matmul(a, b);
```
makes styles consistent. https://google.github.io/styleguide/cppguide.html#Variable_Names